app/views/arts/create.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Arts#create</h1>
+<p>Find me in app/views/arts/create.html.erb</p>
```

Comment:
There are no views for create, it's a post request

Reply:
I will delete it, part of the automatically generated controller